Debug App/Sources/View Controllers/New UI/StripeAchHeadless/MerchantHeadlessStripeAchFieldsView.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  MerchantHeadlessStripeAchFieldsView.swift
+//  Debug App
+//
+//  Created by Stefan Vrancianu on 03.05.2024.
+//  Copyright © 2024 Primer API Ltd. All rights reserved.
+//
+
+import UIKit
+import SwiftUI
+import PrimerSDK
+
+struct StripeAchFieldsView: View {
+    @ObservedObject var viewModel = StripeAchFieldsViewModel()
+    var onSubmitPressed: () -> Void
+    
+    var body: some View {
+        VStack {
+            Text("Stripe ACH session")
+                .font(.title)
+                .padding(.bottom, 20)
+                .addAccessibilityIdentifier(identifier: AccessibilityIdentifier.StripeAchUserDetailsComponent.title.rawValue)
+            
+            VStack(alignment: .leading) {
+                TextField("First name", text: $viewModel.firstName)
```

Comment:
Should this be localized?

Reply:
This is the example app. I didn't think it was needed. Should it be?